.github/workflows/OS-UnitTests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
     - name: Install Project Packages
       run: |
-        julia --project=@. -e 'using Pkg; Pkg.instantiate(); Pkg.build()'
+        julia --project=@. -e 'using Pkg; Pkg.instantiate()'
```

Comment:
Just out of curiosity, why no `precompile` here?

Reply:
we use the system image for these tests, which invokes pre-compilation